odoo/addons/base/models/res_company.py

```diff
@@ -158,6 +159,11 @@ def _compute_logo_web(self):
             img = company.partner_id.image_1920
             company.logo_web = img and base64.b64encode(tools.image_process(base64.b64decode(img), size=(180, 0)))
 
+    @api.depends('partner_id.image_1920', 'logo')
+    def _compute_uses_default_logo(self):
```

Comment:
shouldn't we have `uses_default_logo` set to True if `company.logo` is False ?
(otherwise it will show a broken image in the template because most of the conditions are `t-if="not res_company.uses_default_logo"`)

Reply:
if company.logo is false the back-end route returns the ODOO logo as a fallback. But in the context of emails I guess people probably wouldn't want that to be sent either.. It's just that I can't re-use this outside of emails then since in other cases we might still want some placeholder.

Reply:
@std-odoo Hey, should be good for another go :)

I did modify the condition by the way (and removed 2 changes outside of mail)